### PR TITLE
en: update security config requirement in source configuration file

### LINKED ISSUE #607: security的配置不明
File: [/zh/source-configuration-file.md](https://docs.pingcap.com/zh/tidb-data-migration/dev/source-configuration-file)

```
 security:                       # 上游数据库 TLS 相关配置                             
    ssl-ca: "/path/to/ca.pem"
    ssl-cert: "/path/to/cert.pem"
    ssl-key: "/path/to/key.pem"
```

文档没有说明这些`/path/to`是在哪个component上的的path. 是dm master的path， 还是dm worker的path, 还是别的instance的path?

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### What is changed, added, or deleted? (Required)

Add a description of the requirement of security config in source-configuration file.md to close #607.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/655
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Liuxiaozhen12** [comment]
/needs-cherry-pick-release-2.0

File: en/source-configuration-file.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ This section describes each configuration parameter in the configuration file.
 | `port` | Specifies the port of the upstream database. |
 | `user` | Specifies the username of the upstream database. |
 | `password` | Specifies the user password of the upstream database. It is recommended to use the password encrypted with dmctl. |
-| `security` | Specifies the TLS config of the upstream database. |
+| `security` | Specifies the TLS config of the upstream database. The configuration file path of the certificates need to be accessed by all nodes. If the file path is local, then all the nodes in the cluster need to copy the certificates in the same path of each host.|
```
Comment (CharLotteiu) [inline en/source-configuration-file.md:73]:
```suggestion
| `security` | Specifies the TLS config of the upstream database. The configured file paths of the certificates must be accessible to all nodes. If the configured file paths are local paths, then all the nodes in the cluster need to store a copy of the certificates in the same path of each host.|
```

**Review by CharLotteiu — COMMENTED** [review]
rest LGTM

**Review by lance6716 — APPROVED** [review]

**Review by CharLotteiu — APPROVED** [review]
LGTM

**Pull request merged**